pytest_pootle/fixtures/views.py

```diff
@@ -132,11 +135,21 @@ def project_views(request, client):
 
 
 @pytest.fixture(params=TP_VIEW_TESTS.keys())
-def tp_views(request, client):
+def tp_view_test_names(request):
+    return request.param
+
+
+@pytest.fixture(params=(None, 'member', 'admin'))
```

Comment:
the way that @ta2-1 has implemented it preserves the test names for the parametrized tests.